tox.ini

```diff
@@ -39,7 +39,7 @@ deps =
     latest: Django==4.1.*
     latest: graphene-django==3.0.*
 depends = coverage_setup
-commands = coverage run -m pytest
+commands = coverage run -m pytest -Wall
```

Comment:
To show the problem: https://github.com/browniebroke/django-countries/actions/runs/4881914261/jobs/8711367341#step:5:50

Let me know if you'd rather have it configured via pytest config in `setup.cfg`/`pyproject.toml`.